README.md

```diff
@@ -99,6 +99,15 @@ And see the `X-Envoy-Peer-Metadata-Id` is set to the envoy sidecar id:
 }
 ```
 
+The `X-Envoy-Peer-Metadata-Id` is the Istio HTTP metadata exchange header. These headers will be used for demo purposes, but Istio can be configured to strip the `x-envoy-metadata` headers for requests to service outside the mesh. You can disable this by configuring the meshConfig when installing Istio with:
```

Comment:
I found "this" in "disable this" to be confusing. It read like maybe a double negative to me, so I thought this was clearer?
```suggestion
The `X-Envoy-Peer-Metadata-Id` is the Istio HTTP metadata exchange header. These headers will be used for demo purposes, but Istio can be configured to strip the `x-envoy-metadata` headers for requests to services outside the mesh. You can configure this in the meshConfig by installing Istio with:
```

Reply:
Yeah that sounds fine!